doc/Manuals/ig-ss_x-road_v6_security_server_installation_guide.md

```diff
@@ -553,6 +556,61 @@ Sometimes, after using `sudo apt-get upgrade` command, some of the packages are
+ii  xroad-addon-metaservices        7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 all          X-Road AddOn: metaservices
+ii  xroad-addon-proxymonitor        7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 all          X-Road AddOn: proxy monitoring metaservice
+ii  xroad-addon-wsdlvalidator       7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 all          X-Road AddOn: wsdlvalidator
+ii  xroad-base                      7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 amd64        X-Road base components
+ii  xroad-confclient                7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 amd64        X-Road configuration client components
+ii  xroad-database-local            7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 all          Meta-package for X-Road local database dependencies
+ii  xroad-monitor                   7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 all          X-Road monitoring service
+ii  xroad-proxy                     7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 all          X-Road security server
+ii  xroad-proxy-ui-api              7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 all          X-Road proxy UI REST API
+ii  xroad-securityserver            7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 all          X-Road security server
+ii  xroad-signer                    7.0.0-0.20210518101436git2ce7deb.ubuntu18.04 amd64        X-Road signer component
+```
+
+The following packages are available in the repository.
+
+```
+root@test-ss:~# apt-cache madison xroad-securityserver
+xroad-securityserver | 7.3.0-0.20210518130325gitc1ecc89.ubuntu18.04 | file:/xroad  Packages
+xroad-securityserver | 7.1.0-0.20210518125551git4c60709.ubuntu18.04 | file:/xroad  Packages
+```
+
+Now trying to upgrade the central server packages directly will produce the following error.
+
+```
+root@test-ss:~# apt-get upgrade xroad-securityserver
+...
+Preparing to unpack .../0-xroad-securityserver_7.3.0-0.20210518130325gitc1ecc89.ubuntu18.04_all.deb ...
+This package can be upgraded up to version 7.2.x
+```
+
+The fix is to upgrade the central server in two separate steps. First upgrade to 7.1.x with the following command.
+
+```
+apt install xroad-securityserver=7.1.0-0.20210518125551git4c60709.ubuntu18.04 xroad-proxy=7.1.0-0.20210518125551git4c60709.ubuntu18.04 xroad-monitor=7.1.0-0.20210518125551git4c60709.ubuntu18.04 xroad-addon-metaservices=7.1.0-0.20210518125551git4c60709.ubuntu18.04 xroad-addon-messagelog=7.1.0-0.20210518125551git4c60709.ubuntu18.04 xroad-addon-proxymonitor=7.1.0-0.20210518125551git4c60709.ubuntu18.04 xroad-addon-wsdlvalidator=7.1.0-0.20210518125551git4c60709.ubuntu18.04 xroad-proxy-ui-api=7.1.0-0.20210518125551git4c60709.ubuntu18.04 xroad-confclient=7.1.0-0.20210518125551git4c60709.ubuntu18.04 xroad-signer=7.1.0-0.20210518125551git4c60709.ubuntu18.04 xroad-database-local=7.1.0-0.20210518125551git4c60709.ubuntu18.04 xroad-base=7.1.0-0.20210518125551git4c60709.ubuntu18.04
```

Comment:
The official NIIS package repositories have separate repositories tracking a specific version (e.g. `deb https://artifactory.niis.org/xroad-release-deb bionic-7.1.0 main`), so an alternative is to use that when upgrading instead of "current", avoiding listing all the versions and packages separately.

Reply:
Fixed in https://github.com/nordic-institute/X-Road/pull/957/commits/1a68404ce820ee38ef4dfb67d09a108a71a8f74c